documentation/source/users/rmg/kinetics.rst

```diff
@@ -24,26 +24,36 @@ parameters
 +-------+------------------------------------------------------+
 |Rank   |Example methods                                       |
 +=======+======================================================+
-|Rank 1 |Experiment                                            |
+|Rank 1 |Experiment/FCI                                        |
 +-------+------------------------------------------------------+
-|Rank 2 |High level calculation: CCSD(T)-F12, MS-Tor, 2d rotors|
+|Rank 2 |W4/HEAT with very good (2-d if necessary) rotors      |
 +-------+------------------------------------------------------+
-|Rank 3 |Mid level calculation:  CBS-QB3, 1-d rotors           |
+|Rank 3 |CCSD(T)-F12/CBS with good (2-d if necessary) rotors   |
 +-------+------------------------------------------------------+
-|Rank 4 |Low level calculation: no rotor treatment             |
+|Rank 4 |CCSD(T)-F12/D2,                                       |
 +-------+------------------------------------------------------+
-|Rank 5 |User's estimates without supporting methods           |
+|Rank 5 |CBS-QB3 with 1-d rotors                               |
 +-------+------------------------------------------------------+
-|Rank 10|Averaged value from child nodes' rate rules           |
+|Rank 6 |Double-hybrid DFT with 1-D rotors                     |
 +-------+------------------------------------------------------+
-|Rank 0 |Untrusted method and never used in generation         |
+|Rank 7 |Hybrid DFT (w/ dispersion) (no rotors)                |
++-------+------------------------------------------------------+
+|Rank 8 |B3lyp & lower DFT (no rotors)                         |
++-------+------------------------------------------------------+
+|Rank 9 |Group Additivity                                      |
++-------+------------------------------------------------------+
+|Rank 10|Direct Estimate/Guess                                 |
++-------+------------------------------------------------------+
+|Rank 11|Average of Rates                                      |
++-------+------------------------------------------------------+
+|Rank 0 |General Estimate (Never used in generation)           |
 +-------+------------------------------------------------------+
 
 The rank of 0 is assigned to kinetics that are generally default values for top level nodes 
 that we have little faith in.  It is never used in generation and its value will in fact be overriden
 by averages of its child nodes, which generates an averaged rate rule with rank 10.  
```

Comment:
This is no longer updated (rank 10 for average)

Reply:
Strange that both `0` and `11` are considered inaccurate...

Reply:
fixed